Python/accumulate_position_estimates/accumulate_position_estimates.py

```diff
@@ -0,0 +1,70 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Position:
+    x: float = 0
+    y: float = 0
+    z: float = 0
+
+
+pool_size = Position(25, 12.5, 5)
+
+
+class PositionEstimate:
+    def __init__(self, m_last_elem, distance_threshold) -> None:
+        self.mean = Position(0, 0, 0)
+        self.variance = Position(0, 0, 0)
+        self.measurements = [[], [], []]
+
+        self.m_last_elements = m_last_elem
+        self.distance_threshold = distance_threshold
+
+    def integrate_new_measurement(self, measurment: list):
+
+        if (
+            measurment[0] < pool_size.x
+            and measurment[1] < pool_size.y
+            and measurment[2] < pool_size.z
+        ):
```

Comment:
probably better to make a function as in `check_pool_boundaries`. We can most likely throw away the negative outliers as well.

And it's `measurement` not `measurment` :)